Trappist/src/Promact.Trappist.Repository/TestConduct/ITestConductRepository.cs

```diff
@@ -35,5 +35,18 @@ public interface ITestConductRepository
         /// <param name="testLink">link to conduct a particular test</param>
         /// <returns>instructions for a particular test</returns>
         Task<TestInstructionsAC> GetTestInstructionsAsync(string testLink);
+        
+        /// <summary>
+        /// Stores answers as Key-Value pair in the Database
```

Comment:
store answers as Key-Value pair in the Database.
we use the singular in a case of a bunch of objects.